src/features/retrospective/ui/RetrospectiveDetailPopover.tsx

```
@@ -0,0 +1,148 @@
+  onClose,
+}) => {
+  const { data: expenses } = useExpenses({
+    period: { year: 2025, month: 4 },
+    consumptionKind,
+  });
+
+  const [totalAmount, setTotalAmount] = useState<number>(0);
+  const [categories, setCategories] = useState<Category[]>([]);
+  const [selectedCategories, setSelectedCategories] = useState<Category[]>([]);
+
+  const { data: filteredExpenses, isLoading } = useExpenses({
```

ESLint

[error] 'isLoading' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)